src/test/resources/Features/OCD-3159-scheduledJobs.feature

```diff
@@ -0,0 +1,40 @@
+@Regression
+Feature: OCD-3159 - Types of Jobs listed in Scheduled Jobs page
+  This test checks whether the correct jobs are listed in scheduled jobs page based on user role
```

Comment:
the scenario that is added might be good as a regression test to check all jobs are available/displayed to specific user (currently only ADMIN; ONC access may be added as a part of this ticket or later and then this test will need an update),  but that is not what the ticket is for. For this ticket, relevant test is to check scheduled system job triggers display on screen (after it's scheduled). If the goal is to check access based on role then adding a test that ROLE ACB does not see any of the systemJobs is a good idea (test that simply looks for systemJobs text not found on page).